test/go-test.el

```diff
@@ -0,0 +1,14 @@
+(require 'codemetrics)
+(require 'go-mode)
+
+(add-to-list 'load-path "test")
+(require 'utils)
+
+(codemetrics-test simple-go-code
+                  "test/go/Simple.go"
+                  go-mode
+                  '(2
+                    (function_declaration . 0)
+                    (if_statement . 1)
+                    (call_expression . 0)
+                    (for_statement . 1)))
```

Comment:
Can you add basic header and footer?

Reply:
You mean the comment header and footer? Added those to both tests in latest commits. Put you as the maintainer on the copyright.

Reply:
You can put yourself down if you want to! :)